crates/modelardb_compression/src/models/mod.rs

```diff
@@ -49,10 +47,10 @@ pub(super) const VALUE_SIZE_IN_BYTES: u8 = mem::size_of::<Value>() as u8;
 /// Size of [`Value`] in bits.
 pub(super) const VALUE_SIZE_IN_BITS: u8 = 8 * VALUE_SIZE_IN_BYTES;
 
-/// General error bound that is guaranteed to not be negative, infinite, or NAN.
-/// For [`PMCMean`] and [`Swing`] the error bound is interpreted as a relative
-/// per value error bound in percentage, while [`Gorilla`] uses lossless
-/// compression.
+/// General error bound that is guaranteed to not be negative, infinite, or NAN. For [`PMCMean`],
+/// [`Swing`], and [`Gorilla`],  the error bound is interpreted as a relative per value error bound
```

Comment:
Minor but it seems there is one extra space after `[Gorilla],`.

Reply:
Fixed.